src/cookbook/index.md

```diff
@@ -103,6 +103,7 @@ reference to help you build up an application.
 
 
 ## Plugins
+- [Add achievements and leaderboards to your mobile game]({{site.url}}/cookbook/plugins/achievements-leaderboard)
```

Comment:
@filiph, I thought you were going to add a new "## Games" section rather than hide this in "Plugins"? I'd like to see it in Games.

Reply:
Will do!